rest/model/conversions.go

```diff
@@ -178,6 +214,15 @@ func stripPackage(pkg string) string {
 	return split[len(split)-1]
 }
 
+// stripArray takes a type name and strips the array symbol, returning
+// whether the type was an array as well as the array element's type
```

Comment:
Nit: comments should be sentences, since the comment continues on 219 this should end with punctuation. Also thanks for including examples!

Reply:
added a period :)